com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [2.3.1] - 2022-12-06
+## [2.3.1] - 2022-03-02
 ### Added
 - At import, additional data is added to GameObject cameras to conform with URP or HDRP rendering pipelines.
 ### Changed
```

Comment:
This section is empty, it should be removed ideally. I see that it was kept for the other releases, but I'm not sure it's really the right way to do it.

I'll let Tech Writer to give their opinion on the matter. But I vote to remove this empty section!

Reply:
I am also pro-Remove the title for the Empty section

Reply:
Before December 2020, empty sections were removed. I will check in other packages.